tests/server/api/routers/v1/test_dataset_examples.py

```diff
@@ -0,0 +1,165 @@
+from strawberry.relay import GlobalID
+
+
+async def test_get_dataset_examples_404s_with_nonexistent_dataset_id(test_client):
+    global_id = GlobalID("Dataset", str(0))
+    response = await test_client.get(f"/v1/datasets/{global_id}/examples")
+    assert response.status_code == 404
+    assert response.content.decode() == f"No dataset with id {global_id} can be found."
+
+
+async def test_get_dataset_examples_404s_with_invalid_global_id(test_client, simple_dataset):
```

Comment:
`simple_dataset` need in this test case?

Reply:
yeah, because I want the object to actually exist, and only have the ID malformed, otherwise the test is superficial in a sense